src/remote_adapter/remote_adapter_client/qt_impl/sdl_remote_adapter_qt_client.h

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <QtCore>
+#include <QObject>
+
+#include <string>
+#include <utility>
+
+#include "sdl_remote_adapter_client.h"
+// #include "qt_impl/sdl_remote_adapter_receive_thread.h"
```

Comment:
@aderiabin if this include is redundant please remove

Reply:
@EKuliiev Unused code was removed (see bc3f86577b740bc84201d7d0edaf82cce1d8ec52)